Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.py

```diff
@@ -0,0 +1,854 @@
+    :return: global_msg, incorrect_fields
+    """
+    incorrect_fields = [i for i in populate_fields if i not in incidents_df.columns.tolist()]
+    if incorrect_fields:
+        global_msg += "%s \n" % "%s might not be corect spelling. Please correct or ignore this message" % ' , '.join(
+            incorrect_fields)
+    return global_msg, incorrect_fields
+
+
+def return_outputs_similar_incidents_empty():
+    """
+    Return entry and context for similar incidents if no similar incidents were found
+    :return:
+    """
+    hr = '### Similar Incident' + '\n'
+    hr += 'No Similar incident were found.'
+    return_outputs(readable_output=hr,
+                   outputs={'DBotFindSimilarIncidents': create_context_for_incidents()})
+
+
+def enriched_with_indicators_similarity(full_args_indicators_script: Dict, similar_incidents: pd.DataFrame):
+    """
+    Take DataFrame of similar_incidents and args for indicators script and add information about indicators
+    to similar_incidents
+    :param full_args_indicators_script: args for indicators script
+    :param similar_incidents: DataFrame of incidents
+    :return: similar_incidents enriched with indicators data
+    """
+    indicators_similarity_json = get_similar_incidents_by_indicators(full_args_indicators_script)
+    indicators_similarity_df = pd.DataFrame(indicators_similarity_json)
+    if indicators_similarity_df.empty:
+        indicators_similarity_df = pd.DataFrame(columns=['similarity indicators', 'Identical indicators', 'type', 'id'])
+    keep_columns = [x for x in indicators_similarity_df.columns if x not in similar_incidents]
+    indicators_similarity_df.index = indicators_similarity_df.id
+    similar_incidents = similar_incidents.join(indicators_similarity_df[keep_columns])
+    values = {'similarity indicators': 0, 'Identical indicators': "", 'type': ""}
+    similar_incidents = similar_incidents.fillna(value=values)
+    return similar_incidents
+
+
+def main():
+    similar_text_field, similar_json_field, similar_categorical_field, exact_match_fields, display_fields, from_date, \
+        to_date, show_distance, confidence, max_incidents, query, aggregate, limit, show_actual_incident, \
+        incident_id, include_indicators_similarity = get_args()
+
+    global_msg = ""
+
+    populate_fields = similar_text_field + similar_json_field + similar_categorical_field + exact_match_fields \
+        + display_fields + ['id']
+    populate_high_level_fields = keep_high_level_field(populate_fields)
+
+    incident, incident_id = load_current_incident(incident_id, populate_high_level_fields)
+    if not incident:
+        global_msg += "%s \n" % MESSAGE_NO_CURRENT_INCIDENT %incident_id
+        return_outputs_summary(confidence, 0, 0, [], global_msg)
+        return_outputs_similar_incidents_empty()
+        return
+
+    # load the related incidents
+    populate_fields = display_fields + similar_text_field + similar_json_field + similar_categorical_field \
```

Comment:
for maintenance might be better not to declare `populate_fields` twice, you can remove `id`  from existing list if needed